### ISSUE TITLE
Flatpak version of PolyMC has broken audio

### ISSUE THREAD

**Issue by mutageneral** [PolyMC/PolyMC#733]
### Operating System

Linux

### Version of PolyMC

PolyMC 1.3.1-d7fad4bd Flatpak and Appimage.

### Description of bug

**TL;DR: The sound of Minecraft 1.18.2 in the Flatpak version of PolyMC with PipeWire will do pop and crackle noises, but not the AppImage.**

I am on Steam Deck and when I launch Minecraft 1.18.2, the menu music has pop and crackle noises and every time I hear one, the number of `ERR` (Errors) in `pw-top` goes up:
![image](https://user-images.githubusercontent.com/62779748/172259829-fc57d785-3bb2-4fd1-b4db-935402bd3372.png)

This does not happen in Minecraft 1.6.4 when I tried it (much older version):
![image](https://user-images.githubusercontent.com/62779748/172259376-4918c127-5f20-4de5-b640-9138dee4d435.png)

In `pw-top` I noticed the Quantum of Minecraft 1.6.4 was `1024` and Minecraft 1.18.2 was `512`. This might be causing **xrun** errors and so the pop and crackle noises?

Definition of "xrun":
> An "xrun" can be either a buffer underrun or a buffer overrun. In both cases an audio app was either not fast enough to deliver data to the ALSA audio buffer or not fast enough to process data from the ALSA audio buffer. Usually xruns are audible as crackles or pops.

Then, I installed the flatpak PolyMC on my laptop running Fedora (PipeWire) and the exact same sound problems occures with Minecraft 1.18.2. (forgot to take a screenshot, but it looks the same as the first screenshot). **This means this is not a Steam Deck specific problem.**

Then I installed the PolyMC **AppImage** to see if it would change something. Now there are **no more sounds issues** and **the `ERR` (Errors) number in `pw-top` is not increasing** while the game is running:
![image](https://user-images.githubusercontent.com/62779748/172260382-dcf1f0cb-98c9-44ba-9c9b-65138ad653ea.png)

### Steps to reproduce

* Use a GNU/Linux distribution with Flatpak and Pipewire installed
* Install Flatpak version of PolyMC
* Log in and Install Minecraft 1.18.2
* Open the game and listen for pops and crackles. You can also open `pw-top` and look at the `ERR` (Errors) going up (not normal behaviour)

### Suspected cause

Something to do with Pipewire shenanigans, Flatpak containerization and the fact the new Minecraft versions use a smaller quantum value than the older versions.

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by mutageneral** [comment]
Seems like this was already reported in the Flathub PolyMC repo with less information than I provided by @logan2611: https://github.com/flathub/org.polymc.PolyMC/issues/22

**Comment by mutageneral** [comment]
For anyone wondering on how to use the PolyMC AppImage version on Steam Deck, here is what I did:

* Downloaded the Flatpack version of PolyMC to get a version of java installed.
* Downloaded the PolyMC AppImage and went thought the normal configuration and log in.
* When prompted that no java version was detected, open the file picker and go to this location:  `/var/lib/flatpak/app/org.polymc.PolyMC/current/active/files/jdk/17/bin/java`
* Add the Flatpak as a non-Steam game and go back to gaming mode.

Also, using the AppImage version means you don't get bug #396, a bug where you can't open file pickers on Steam Deck because Valve forgot to add some Flatpak related things.

**Comment by d-513** [comment]
I have no idea why this is happening but indeed has been reported. Can you check official launcher flatpak for these issues?

**Comment by logan2611** [comment]
I can't test with the official launcher without migrating my account unfortunately, but this issue is also present in GDLauncher.

**Comment by xVANKO** [comment]
I confirm this issue - no sound at all in current PolyMC version on Linux

**Comment by mutageneral** [comment]
> I confirm this issue - no sound at all in current PolyMC version on Linux

No sound at all? That's strange. What does `pw-top` looks like while the game is running?
